clippy_lints/src/casts/mod.rs

```diff
@@ -473,6 +474,28 @@ declare_clippy_lint! {
     "casts from an enum tuple constructor to an integer"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks for uses of the `abs()` method that cast the result to unsigned.
+    ///
+    /// ### Why is this bad?
+    /// The `unsigned_abs()` method avoids panic when called on the MIN value.
```

Comment:
It looks like that the use of `wrapping_abs()` avoids panic. Why this lint catch the use of it?

Reply:
Because now that is available `unsigned_abs()` is nicer than `wrapping_abs() as u32`... It is true that in this case it is just a style matter, but since it was easy to catch I added it. If you prefer to remove it let me know.

Reply:
IMO it would be better to remove `wrapping_abs()` because it confuses users.

Reply:
Okay! done